x-pack/plugins/global_search/server/routes/integration_tests/find.test.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { of, throwError } from 'rxjs';
+import supertest from 'supertest';
+import { UnwrapPromise } from '@kbn/utility-types';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { setupServer } from '../../../../../../src/core/server/test_utils';
```

Comment:
shouldn't we whitelist `test_utils` ?

Reply:
We should (and I thought we already were). I was going to look at this

Reply:
https://github.com/elastic/kibana/blob/bf04235dae35452061cc7ea3d86d96c19a58206c/.eslintrc.js#L223 optional extension is not present. Will fix that in this PR.